scripts/exrrfs_jedivar.sh

```diff
@@ -144,6 +148,19 @@ if [[ ${start_type} == "warm" ]] || [[ ${start_type} == "cold" && ${COLDSTART_CY
   # check the status
   export err=$?
   err_chk
+  if [[ ${start_type} == "warm" ]] && [[ -v SNUDGETYPES ]]; then
+      # pyioda libraries
+      PYIODALIB=$(echo "$HOMErdasapp"/build/lib/python3.*)
+      WXFLOWLIB=${USHrrfs}/wxflow/src
+      export PYTHONPATH="${WXFLOWLIB}:${PYIODALIB}:${PYTHONPATH}"
+      python ${USHrrfs}/snudge.py ${CDATE} ${SNUDGETYPES} ${DATA}/${initial_file} 
+      if [[ ! -s "soil_analyzed.nc" ]]; then
+        echo "Soil nudging failed"
+        exit 1
```

Comment:
If soil nudging fails, we would like to continue the workflow.
But we can output warning messages like `echo "WARNING: soil nudging failed"`